capsules/src/lps25hb.rs

```diff
@@ -191,6 +191,7 @@ impl<'a> i2c::I2CClient for LPS25HB<'a> {
                 buffer[0] = Registers::CtrlReg1 as u8;
                 buffer[1] = 0;
                 self.i2c.write(buffer, 2);
+                self.interrupt_pin.disable_interrupt();
```

Comment:
Should this be in the `Done` state instead and paired up with the `i2c.disable` ?

Reply:
It could certainly be disabled in the Done state. I put it here because that is what the tsl2561 capsule already does. I don't think it matters in practice.